src/op_mode/show_conntrack.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2022 VyOS maintainers and contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import xmltodict
+from tabulate import tabulate
+from vyos.util import cmd
+
+conntrack_data = cmd(f'sudo  conntrack -L -o xml')
+conntrack = xmltodict.parse(conntrack_data)
+
+data_entries = []
+
+for entry in conntrack['conntrack']['flow']:
```

Comment:
While we don't have a final design for the machine-readable op mode yet, the script should have a function for retrieving raw data and a function for formatting that data, rather than a monolithic code block. At the very least having it in separate functions will simplify testing, and when we have a final design, we will not need to rewrite the code to use it—we can just call those functions.

Reply:
Added `get_raw_data()`, `get_formatted_output`